cmd/kube-apiserver/app/server.go

```diff
@@ -35,7 +35,7 @@ import (
 	"github.com/spf13/cobra"
 
 	extensionsapiserver "k8s.io/apiextensions-apiserver/pkg/apiserver"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
```

Comment:
keep original

Reply:
this is autogenerated and does no harm

Reply:
I think this is the last thing. please let me know if I need to go back and update this.